YetiLoader/YetiLoader.cpp

```diff
@@ -67,23 +68,24 @@ int main(int argc, char** argv)
 	}
 
 	// find path to lockdown, and prompt user if we cant find it
-	auto ld_exe = L"C:\\Program Files (x86)\\Respondus\\LockDown Browser\\LockDownBrowser.exe";
+	auto ld_exe = L"C:\\Program Files (x86)\\Respondus\\LockDown Browser\\LockDownBrowserOEM.exe";
```

Comment:
I believe that a majority of users use the regular Lockdown Browser, not the lab edition. Could you switch this?